newcomers-tutorial.md

```diff
@@ -98,7 +95,8 @@ cd ghc/
   on the compiler itself; without it, a release build will be made, which takes
   significantly longer and is less convenient for development.
 
-Now go make yourself some coffee while the build runs.
+Now go make yourself some coffee while the build runs. Don't be too worried if it
+takes around 20 minutes or more for your first build.
```

Comment:
Yeah. I realised that when I wrote it, but I was thinking of the poor soul (such as myself on running my first build) who would be worried that something was wrong, and stop the process, then restart it several times, only to find out that 30 minutes for the first build isn't unheard of or crazy. What are your suggestions for that? That's why I said don't be too worried. It's not a hard number, it's a soft number.

I guess this kind of comes down to... what kind of people are we onboarding? Are they 10 year haskell devs? 2 year haskell devs? 2 month haskell devs? My assumption for new people is we usually want to make it as smooth as possible for as many people who want to contribute as can... because the more people we get who can contribute the merrier. But maybe that's not the general way things are? I don't know. I'm new to this :)

Reply:
I'll adjust so it says "if it takes a while". It just is a bit vague... and my personal definition of 'a while' is a lot less time than 20 minutes. I was a bit shocked when it took over 30 minutes for my first build, TBH. (But that's because I didn't realise how *big* haskell was yet then)